packages/common-all/src/types/configs/publishing/giscus.ts

```diff
@@ -0,0 +1,77 @@
+// https://github.com/giscus/giscus-component/blob/main/react/src/lib/types.ts
```

Comment:
How should I go about properly attributing the project here?

They don't have a separate type package that I could use and it's a bit weird to add `@giscus/react` to `common-all` just for the types.